src/webview/static/base.css

```diff
@@ -475,3 +475,30 @@ blockquote {
   height: 32px;
   fill: hsla(0, 0%, 100%, 0.75);
 }
+.edit-history-block{
+  padding:8px;
+  border-bottom:1px solid hsla(0, 0%, 50%, 0.25);
+}
+.edit-history-block .static-timestamp{
+  margin-top: 8px;
+  text-align: right;
+}
+.edit-history-content{
+  -webkit-user-select: text; /* Safari 3.1+ */
+  -moz-user-select: text; /* Firefox 2+ */
+  -ms-user-select: text; /* IE 10+ */
+  user-select: text; /* Standard syntax */
+  -khtml-user-select: text;
+  -webkit-touch-callout: text;
+  word-wrap: break-word;
+}
+.highlight_text_inserted {
+  color: #9effa1;
+  background-color: hsla(120,64%,95%,.3);
+}
+.highlight_text_deleted {
+  color: red;
+  background-color: #fcdcd8;
+  text-decoration: line-through;
+  word-break: break-all;
+}
```

Comment:
Are these based on something in the webapp, or did you develop them afresh?

Either is fine :slightly_smiling_face: -- I'd just like to know (and have the commit message say so for future readers), and if it's from the webapp have a quick comment in the file saying where exactly it's from. (Including if it's based loosely or in part on something in the webapp.) That will save us some archaeological work in any future effort to unify the message CSS on web and mobile.